pkg/auth/providers/azure/azure_actions.go

```diff
@@ -120,7 +132,14 @@ func (ap *azureProvider) migrateToMicrosoftGraph() error {
 		return err
 	}
 	if authProviderEnabled(cfg) {
-		updateAzureADConfig(cfg)
+		updateAzureADEndpoints(cfg)
+
+		if cfg.ObjectMeta.Annotations == nil {
+			cfg.ObjectMeta.Annotations = map[string]string{GraphEndpointMigratedAnnotation: "true"}
+		} else {
+			cfg.ObjectMeta.Annotations[GraphEndpointMigratedAnnotation] = "true"
+		}
```

Comment:
nit: because I don't love else statements.
```suggestion
		if cfg.ObjectMeta.Annotations == nil {
			cfg.ObjectMeta.Annotations = map[string]string{}
		}
		cfg.ObjectMeta.Annotations[GraphEndpointMigratedAnnotation] = "true"
```

Reply:
Good point!